docs/design/generics/terminology.md

```diff
@@ -637,60 +637,54 @@ conditions on the type argument. For example: `Array(T)` might implement
 
 ## Interface type parameters and associated types
 
-Imagine an interface defining a container. Different containers will contain
-different types of values, and the container API will have to refer to that
-"element type" when defining the signature of methods like "insert" or "find".
-If that element type is a parameter (input) to the interface type, we say it is
-an _interface type parameter_; if it is an output, we say it is an _associated
-type_. An associated type is a kind of [associated entity](#associated-entity).
-
-Interface type parameter example:
-
-```
-interface StackTP(ElementType:! Type)
-  fn Push[addr me: Self*](value: ElementType);
-  fn Pop[addr me: Self*]() -> ElementType;
-}
-```
-
-Associated type example:
+_Interface type parameters_ and _associated types_ are both ways of allowing the
+types in the signatures of function in an interface to vary. For example,
```

Comment:
```suggestion
types in function signatures in an interface to vary. For example,
```

Note "function" should have been pluralized, but I think "function signatures" is easier to read.